azext_edge/__init__.py

```diff
@@ -6,20 +6,38 @@
 
 from azure.cli.core import AzCommandsLoader
 from azext_edge.constants import VERSION
+from knack.events import EVENT_INVOKER_POST_PARSE_ARGS
+
+
+def version_check_handler(cli_ctx, **kwargs):
+    from .edge.util.version_check import check_latest
+
+    command: str = kwargs.get("command")
+    if command:
+        if command.startswith("iot ops"):
+            if command == "iot ops init":
+                ensure_latest = kwargs["args"].ensure_latest
```

Comment:
Just a thought - should we also check `'ensure_latest' in kwargs['args']` before this accessor, in case we rename/move/remove this param in `params.py`?

Reply:
Good thought. I think we should punt for a subsequent iteration. As you mentioned in another setting, init should should throw if the arg changes.